CRM/Contact/Form/Task/PDFTrait.php

```diff
@@ -232,10 +213,7 @@ public function getFileName(): string {
    *     if the form controller does not exist), else FALSE
    */
   protected function isLiveMode(): bool {
-    // CRM-21255 - Hrm, CiviCase 4+5 seem to report buttons differently...
-    $buttonName = $this->controller->getButtonName();
```

Comment:
@deb1990 just pinging you to check this change (simplification of detect preview) doesn't affect you. It looks like a very only civicase/civihr work around that is probably obsolete but worth checking...